tools/AzureRM.BootStrapper/AzureRM.BootStrapper.Tests.ps1

```diff
@@ -0,0 +1,231 @@
+Import-Module -Name AzureRM.Bootstrapper
```

Comment:
I added a script that will import pester and bootstrapper modules and run the unit tests.